Check the compatibility between Thrust in CUDA Toolkit and any bundled CUB.
Ignoring this check is just a bad idea, the build will fail anyway if those
are not compatible.
author: Alex Domingo (Vrije Universiteit Brussel)
diff --git a/CMakeLists.txt.orig b/CMakeLists.txt
index 4038ffd..a23571d 100644
--- a/CMakeLists.txt.orig
+++ b/CMakeLists.txt
@@ -74,7 +74,6 @@ if (CMAKE_CUDA_COMPILER)
 	enable_language(CUDA)
 	set_property(TARGET lisflood PROPERTY CUDA_SEPARABLE_COMPILATION ON)
 	
-	add_compile_definitions(CUDA THRUST_IGNORE_CUB_VERSION_CHECK)
 	target_sources(lisflood PRIVATE
 		cuda/cuda_boundary.cu
 		cuda/cuda_dem.cu
